### ISSUE TITLE
PKG installer on Mac isn't good yet.

### ISSUE THREAD

**Issue by JezSan** [ethcore/parity#4913]
Quite a few issues with the Mac version, especially when installed from the Installer pkg!

1.  Parity doesn't work on Safari (at all!) when installed from the Installer (how was this not obvious during testing!?)

This is because scrolling doesn't work, and the user license agreement that appears when installed from the installer, doesn't let you scroll, thus you cant click next, thus you can never get beyond the user agreement.  Rather Frustrating... so near, and yet so far...!

Safari is the standard web browser on the Mac.  If it doesn't work on the standard web browser, thats a big deal, and effectively, it doesn't work on the Mac (...for the people likely to use the Installer).  I know it runs on Chrome and you guys like Chrome, but that has other problems... read on.

2.  Parity toolbar app doesn't give you the option to run Chrome version, so you're stuck with the Safari version.  not a problem if (1) is fixed, but for now, it means Parity isn't usable on a Mac (when installed with the Installer)

3.  You cant run Chrome version on your own, because it isn't signed.  Parity says it needs a token signed with the signer.   But you cant run the Signer because you cant run the command line version of parity if you used the installer (!)...   because...

4.  The installer doesn't set the paths correctly so you cant use Parity from the command line (Terminal).  Thus, you cant run the Signer, to allow Chrome to work.. (see 3).

Notes:  The Installer is intended to make Parity usable by 'regular folk' - people who aren't developers.  these are people (like me) who need it to work out of the box without going into configurations and path settings, and Terminals, and command line options etc.

In short, the Installer caused a cascade of problems that weren't present when you run the Brew version.  The Brew installed version works great!!   Hopefully you can make the installer a little better so that Parity installed via the Installer also runs great as well.

Keep up the good work.  despite the problems with the installer, Parity is an excellent piece of work.

**Comment by 5chdn** [comment]
Already fixed, get the latest insaller here: https://github.com/ethcore/parity/releases/tag/v1.6.3

**Comment by debris** [comment]
@5chdn can we also close #3274 #3275 #3276 #3699 ?

**Comment by 5chdn** [comment]
@debris will look into them, thanks

**Comment by BluBeast** [comment]
Installed parity on brand new MacBook Pro (it ended up on Safari)....it accepted ETH in the account but won't move around Parity or let the cursor move around. It's stuck on the account/wallet.  Should I reinstall, try to put it on Chrome, or try something else?  Home Brew is on the system now.

**Comment by 5chdn** [comment]
Your mouse is stuck? Could you clarify what issue you experience? Does this only happen in the Parity wallet?
